### ISSUE TITLE
Plugin crashes on ios 12

### ISSUE THREAD

**Issue by Tallyb** [BranchMetrics/capacitor-branch-deep-links#59]
Although it is mentioned that this plugin is working with ios 12 
https://github.com/BranchMetrics/capacitor-branch-deep-links/blob/master/ios/Podfile#L1
it may in fact crash on ios 12. 

The reason being this line: https://github.com/BranchMetrics/capacitor-branch-deep-links/blob/master/ios/Podfile#L7 
In certain cases, and depending on cache, it may install Branch 1.42.0 which is not compatible with ios 3. 

We fixed this (after 3 days of working on it...) by adding the Branch 1.4.1 as dependency: 

in the ios/App/Podfile: 

```
target 'app' do
  capacitor_pods
  pod 'Branch', '1.41.0'  # add this line to install the pod good verison that works with ios 12. 
  # Add your Pods here
end
```

Suggestion - to comply with ios12, the POD version on the above Podfile, the version should be fixed on 1.41.0, as 1.42 is no longer compliant. 

Also filed as issue on the Branch framework: https://github.com/BranchMetrics/ios-branch-deep-linking-attribution/issues/1185

**Comment by Deloww** [comment]
https://github.com/BranchMetrics/capacitor-branch-deep-links/issues/59[](https://github.com/BranchMetrics/capacitor-branch-deep-links/issues/59)

**Comment by jf-branch** [comment]
Hi @Tallyb, Closing this out as we have addressed this on the iOS repo: https://github.com/BranchMetrics/ios-branch-deep-linking-attribution/issues/1185 and we have also released 1.43.1 that adds LinkPresentation.framework as a weak dependency in the podspec
